packages/lambda-xyz/src/cli/imagery/regional/auckland.ts

```diff
@@ -1,11 +1,10 @@
 import { EPSG } from '@basemaps/geo';
-import { MosaicCog } from '../../tiff.mosaic';
+import { MosaicCog } from '../mosaic.cog';
```

Comment:
should we just delete all the imagery now? Its not really needed.

Other option is we could just spider it from the s3 folder.

Reply:
Decided on deprecating it now, after the next production release removing it.